src/main/java/io/github/syst3ms/skriptparser/expressions/ExprStringCharAt.java

```diff
@@ -0,0 +1,54 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import io.github.syst3ms.skriptparser.Parser;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+import io.github.syst3ms.skriptparser.util.DoubleOptional;
+
+/**
+ * The character at a given position in a string. Note that indices in Skript start at 1.
+ *
+ * @name Character at
+ * @pattern char[acter] at [(index|pos[ition])] %number% (of|in) %string%
+ * @since ALPHA
+ * @author Olyno
+ */
+public class ExprStringCharAt implements Expression<String> {
+
+	static {
+		Parser.getMainRegistration().addExpression(
+			ExprStringCharAt.class,
+			String.class,
+			true,
+			"char[acter] at [(index|pos[ition])] %integer% (of|in) %string%"
+		);
+	}
+
+	private Expression<Integer> position;
+	private Expression<String> value;
+
+	@SuppressWarnings("unchecked")
+	@Override
+	public boolean init(final Expression<?> @NotNull [] expressions, final int matchedPattern, @NotNull final ParseContext parseContext) {
```

Comment:
Just noticed this. These values should not be annotated as @NotNull. You should remove them all.
If this gives warnings, make sure you have the correct package-info file in the expressions package.